posthog/api/capture.py

```diff
@@ -297,6 +295,7 @@ def drop_events_over_quota(token: str, events: List[Any]) -> List[Any]:
     )
 
     for event in events:
+        # TODO we're quota limiting $heatmap and $performance_event events on the replay bill
```

Comment:
I don't think this is the case anymore now that we include it on general events.

```suggestion
```